compiler/ballerina-tools-api/src/main/java/io/ballerina/tools/text/TextDocument.java

```diff
@@ -50,6 +50,10 @@ public int textPositionFrom(LinePosition linePosition) {
         return lines().textPositionFrom(linePosition);
     }
 
+    public LineMap getLines() {
+        return this.lines();
+    }
+
```

Comment:
Do we need this method?

Reply:
Removed this function since it was not necessary in the newly proposed approach.